addons/website/static/src/components/fields/redirect_field.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <t t-name="website.RedirectField" owl="1">
-        <button type="button" class="oe_stat_button btn btn-light w-100 mb-0 py-0 border-0 border-start border-bottom rounded-0 text-start text-nowrap text-capitalize"
-            t-on-click="onClick"
-            t-att-aria-label="info"
-            t-att-title="info">
-            <i class="fa fa-fw o_button_icon fa-globe" t-att-class="this.props.value ? 'text-success' : 'text-danger'"/>
-            <div class="o_stat_info">
-                <span class="o_stat_text">Go to<br/>Website</span>
-            </div>
-        </button>
-    </t>
+<t t-name="website.RedirectField" owl="1">
```

Comment:
@detrouxdev Except this RedictField template that was not touched by the other commit if I am not mistaken, the other reindentation should be squashed in the related commits where you basically rewrite them entirely anyway. The point of reindenting something in its own commit is to separe concerns in the history. But if we can avoid it... best do it (like taking the opportunity of a file being basically rewritten at 95% to reindent everything: do it there rather than on its own commit).